dali/pipeline/operators/crop/new_crop.h

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_CROP_NEW_CROP_H_
+#define DALI_PIPELINE_OPERATORS_CROP_NEW_CROP_H_
+
+#include <utility>
+#include <vector>
+#include <tuple>
+#include "dali/core/common.h"
+#include "dali/core/error_handling.h"
+#include "dali/pipeline/operators/common.h"
+#include "dali/pipeline/operators/crop/kernel/crop_kernel.h"
+#include "dali/pipeline/operators/crop/crop_attr.h"
+#include "dali/pipeline/operators/operator.h"
+
+namespace dali {
+
+template <typename Backend>
+class NewCrop : public Operator<Backend>, protected CropAttr {
```

Comment:
NewCrop? When we remove the one we have now new won't be new anymore.

Reply:
when we are confident with it we can rename it and remove the old one